.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -225,11 +225,11 @@ run_tests() {
   ansi2html <"/tmp/${LOGFILE}" >"/tmp/${LOGFILE}.html"
   cp -a "/tmp/${LOGFILE}.html" ${ARTIFACT_DIR}/${project}
   cp -a /tmp/backstage-showcase/e2e-tests/playwright-report/* ${ARTIFACT_DIR}/${project}
-  
+
 
   droute_send $release_name $project
 
-  exit ${RESULT}
+  echo ${RESULT}
```

Comment:
We should return and exit as per the result else it will not mark job as failed because of `set -e` while running tests.

Reply:
@subhashkhileri ok it's just for test purpose, let's see if rbac tests will be executed

Reply:
@subhashkhileri do you have any suggestion or can we go with `echo` ?